mongoose.h

```diff
@@ -833,6 +833,24 @@ struct timeval {
 #define MG_ENABLE_PROFILE 0
 #endif
 
+#ifndef MG_ENABLE_TCPIP_DRIVER_INIT    // mg_mgr_init() will also initialize
+#define MG_ENABLE_TCPIP_DRIVER_INIT 0  // enabled built-in driver for
+#endif                                 // Mongoose built-in network stack
+
+#ifndef MG_TCPIP_IP  // e.g. mg_htonl(MG_U32(192, 168, 0, 223))
+#define MG_TCPIP_IP mg_htonl(MG_U32(0, 0, 0, 0))  // or leave as 0 for DHCP
```

Comment:
mg_htonl() is a function, and Keil complains about non-constant initialiser.
Let's create a `MG_IP(a,b,c,d)` macro that creates a number directly in the network byte order?